lib/reservation_manager.rb

```diff
@@ -0,0 +1,136 @@
+require_relative "reservation.rb"
+require "date"
+
+require "awesome_print"
+
+module Hotel
+  class ReservationManager
+    #add constant for room numbers
+    MAX_ROOMS = 20
+    ROOM_RATE = 200
+
+    def initialize
+      @reservations = []
+      @block = []
```

Comment:
If this is an array, it may make sense to make the variable name plural, so `@blocks`